pkg/apis/core/validation/validation.go

```diff
@@ -4071,6 +4072,11 @@ func ValidateService(service *core.Service) field.ErrorList {
 		}
 	}
 
+	//Validate service IPFamily
+	if len(service.Spec.ServiceIPFamily) > 0 && !supportedServiceIPFamily.Has(string(service.Spec.ServiceIPFamily)) {
```

Comment:
we should not need to allow len() = 0 - if it is defaulted, it must be in that list